### PR TITLE
format addscheme and kubeconfig

### PR DESCRIPTION
**Pull request by huiwq1990**
/kind enhancement
> /kind good-first-issue
> /kind feature
> /kind question
> /kind design
> /sig ai
> /sig iot
> /sig network
> /sig storage
> /sig storage

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by huiwq1990** [comment]
@rambohe-ch Need your attention.

File: cmd/yurt-app-manager/app/core.go [hunk 1]
```diff
@@ -53,10 +54,10 @@ var (
 )
 
 func init() {
-	_ = clientgoscheme.AddToScheme(scheme)
 	_ = appsv1alpha1.AddToScheme(clientgoscheme.Scheme)
```
Comment (rambohe-ch) [inline cmd/yurt-app-manager/app/core.go:57]: this line can also be removed?

Comment (huiwq1990) [reply cmd/yurt-app-manager/app/core.go:57]: Currently, it could not be removed, as yurtingress use it. We should refactor the code later.

Comment (rambohe-ch) [reply cmd/yurt-app-manager/app/core.go:57]: ok

**Review by rambohe-ch — APPROVED** [review]

**Pull request closed without merging**